lemur/plugins/lemur_digicert/plugin.py

```diff
@@ -317,10 +319,71 @@ def __init__(self, *args, **kwargs):
 
         self.session.hooks = dict(response=log_status_code)
 
+        # max_retries applies only to failed DNS lookups, socket connections and connection timeouts,
+        # never to requests where data has made it to the server.
+        # we Retry we also covers HTTP status code 406, 500, 502, 503, 504
+        retry_strategy = Retry(total=3, backoff_factor=0.1, status_forcelist=[406, 500, 502, 503, 504])
+        adapter = requests.adapters.HTTPAdapter(max_retries=retry_strategy)
+        self.session.mount("https://", adapter)
+
         super().__init__(*args, **kwargs)
 
-    def get_certificates(self):
-        pass
+    def get_certificates(self, options, **kwargs):
```

Comment:
for backwards compatibility, can we gate this behavior via a config variable e.g. DIGICERT_SOURCE_ENABLED = False and document it in docs/administration.rst as well as adding a note to the changelog on this change?

Reply:
Done!
- Added DIGICERT_SOURCE_ENABLED
- Added documentation
- Added information in CHANGELOG file